Docs/debugging.md

```diff
@@ -0,0 +1,77 @@
+# Debugging
+
+The content of this guide will help you to configure your PHPStorm in order to debug the PIM.
+The following doc is about Linux only and is not tested on other OS.
+
+## Container config
+
+*Xdebug* is deactivated by default. If you want to activate it, you need to set the following environment variables,
+ in this case we use a part of the default configuration of the Akeneo PIM 2.0 which use fpm and httpd but this is the same spirit for others versions:
+```yaml
+services:
+  fpm:
+    image: akeneo/fpm:php-7.1
+    environment:
+      PHP_IDE_CONFIG: 'serverName=pim-ce-cli' # Your will need the server name for your IDE config
+      PHP_XDEBUG_ENABLED: 1 # Set to 0 if you don't want to activate xdebug
+      PHP_XDEBUG_IDE_KEY: 'XDEBUG_IDE_KEY' # the IDE KEY you want to use
+      XDEBUG_CONFIG: 'remote_host=xxx.xxx.xxx.xxx' # Your host IP address, this one is needed only if the docker daemon run on TCP instead of unix domain socket (on Mac OS for instance)
+    volumes:
+      - ./:/srv/pim
+
+  httpd:
+      image: httpd:2.4
+      depends_on:
+        - fpm
+      environment:
+        PHP_IDE_CONFIG: 'serverName=pim-ce' # Your will need the server name for your IDE config
+      ports:
+        - '8080:80' # You will need the 8080 for your configuration
+```
+
+Then you just have to run `docker-compose up -d` again and XDebug will be activated.
+
+## Tools Config
+
+### PHPStorm
+
+First of all, you have to setup your server, in PHPStorm go on the `Setting > Servers` section.
+Then you have to add one, and configure like this.
```

Comment:
setup your **server. In** PHPStorm ... `Setting > Servers` **section, then add a new** one, and configure **it as above**.